CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The intended audience of this file is for py42 consumers -- as such, changes that don't affect
 how a consumer would use the library (e.g. adding unit tests, updating documentation, etc) are not captured here.
 
+## Unreleased
+
+### Fixed
+
+- `sdk.securitydata.search_all_file_events()` automatically escapes double-quote characters in the `page_token` param when passed.
```

Comment:
Someone might get the impression that the the automatic-escaping _was_ the bug that got fixed.

Reply:
I'll clarify.